### PR TITLE
[MI-3818]: Update dialog styles

### PR DESCRIPTION
**Pull request by SaurabhSharma-884**
## Ticket ID
[MI-3818](https://halcyonai.atlassian.net/browse/MI-3818)

## Description
- Update Dialog styles

## Screenshots

Screenshot from 2023-12-06 22-07-45
Screenshot from 2023-12-06 22-08-05
Screenshot from 2023-12-06 22-08-21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@brightscout/mattermost-ui-library",
-	"version": "2.3.1",
+	"version": "2.3.2",
```
Comment (ayusht2810) [inline package.json:3]: update version of lock file as well by running install command

File: src/components/Dialog/Dialog.component.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+	import React from 'react';
```
Comment (ayusht2810) [inline src/components/Dialog/Dialog.component.tsx]: why this change?

File: src/components/Dialog/Dialog.styles.ts [hunk 3]
```diff
@@ -77,7 +79,7 @@ export const DialogActions = styled.div({
 
 // Styles for dialog footer
 export const DialogFooter = styled.div({
-    padding: '18px',
+    padding: '0',
```
Comment (ayusht2810) [inline src/components/Dialog/Dialog.styles.ts]: are we overriding style here?

**Review by ayusht2810 — COMMENTED** [review]
is the screenshot for custom dialog component correct here?

File: src/components/Dialog/Dialog.styles.ts (hunk 3 above)
Comment (SaurabhSharma-884) [reply src/components/Dialog/Dialog.styles.ts]: NO

**Review by ayusht2810 — APPROVED** [review]

File: src/components/Dialog/Dialog.component.tsx [hunk 4]
```diff
@@ -72,13 +72,15 @@ export const Dialog = (props: DialogProps) => {
 			className={`mm-dialog ${className}`}
 			{...restProps}
 		>
-			<DialogHeader>
-				<IconButton onClick={onCloseHandler}>
+			<DialogHeader style={{
+				...(!title && {justifyContent: 'flex-end'})
```
Comment (sooraj-shukla) [inline src/components/Dialog/Dialog.component.tsx]: Use props or utility class.
In cases where we use `styled` and `utility` classes together, all the defined utility classes should use `!important`;

Comment (SaurabhSharma-884) [reply src/components/Dialog/Dialog.component.tsx]: Created a prop

**Review by sooraj-shukla — APPROVED** [review]
